README.md

```diff
@@ -30,6 +30,15 @@ atob('SGVsbG8sIHdvcmxkIQ=='); // 'Hello, world!'
 
 [Per the spec](https://html.spec.whatwg.org/multipage/webappapis.html#atob:dom-windowbase64-btoa-3), `btoa` will accept strings "containing only characters in the range `U+0000` to `U+00FF`." If passed a string with characters above `U+00FF`, `btoa` will return `null`. If `atob` is passed a string that is not base64-valid, it will also return `null`. In both cases when `null` is returned, the spec calls for throwing a `DOMException` of type `InvalidCharacterError`.
 
+## Browsers
+
+If you want to include just one of the methods to save bytes in your client-side code, you could `require` the desired module directly.
+
+```js
+var atob = require('abab/lib/atob')
```

Comment:
Since this repo's style uses semicolons, would you mind adding them to these examples?